### PR TITLE
Formatted setup.py to make it easier to read

### PR DESCRIPTION
**Pull request by krishp058**
I haven't changed any of code, I had just added lines in between of major functions to make them stand out, and to make it easier to navigate.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -67,18 +69,17 @@ def get_scm_config():
                    'hpy/debug/src/dhqueue.c',
                    'hpy/debug/src/_debugmod.c',
                    'hpy/debug/src/autogen_debug_wrappers.c',
-                  ],
+                   ],
```
Comment (antocuni) [inline setup.py:72]: I'm happy to merge PRs to improve readability, but I prefer if we stick to existing conventions. So far all closed brackets are indented at the same level, this breaks the rule

File: setup.py [hunk 2]
```diff
@@ -67,18 +69,17 @@ def get_scm_config():
...
                       'hpy/devel/include',
                       'hpy/universal/src',
                       'hpy/debug/src/include',
                   ],
                   extra_compile_args=[
-                      '-DHPY_UNIVERSAL_ABI',
-                  ] + EXTRA_COMPILE_ARGS
+                                         '-DHPY_UNIVERSAL_ABI',
+                                     ] + EXTRA_COMPILE_ARGS
                   )
```
Comment (antocuni) [inline setup.py:81]: same as above. Compare to e.g. the indentation of the items of `include_dirs`

File: setup.py (hunk 1 above)
Comment (krishp058) [reply setup.py:72]: That's all good!

Comment (hodgestar) [inline setup.py:72]:
```suggestion
                  ],
```

File: setup.py [hunk 3]
```diff
@@ -67,18 +69,17 @@ def get_scm_config():
...
                   include_dirs=[
                       'hpy/devel/include',
                       'hpy/universal/src',
                       'hpy/debug/src/include',
                   ],
                   extra_compile_args=[
-                      '-DHPY_UNIVERSAL_ABI',
-                  ] + EXTRA_COMPILE_ARGS
+                                         '-DHPY_UNIVERSAL_ABI',
+                                     ] + EXTRA_COMPILE_ARGS
```
Comment (hodgestar) [inline setup.py:80]:
```suggestion
                      '-DHPY_UNIVERSAL_ABI',
                  ] + EXTRA_COMPILE_ARGS
```

**Review by hodgestar — APPROVED** [review]
Thanks for the clean-up! I'll merge once the builds are happy.

**Pull request merged**